kernelci/api/latest.py

```diff
@@ -89,37 +89,42 @@ def receive_event(self, sub_id: int):
                 continue
             return event
 
-    def get_node(self, node_id: str) -> dict:
-        return self._get(f'node/{node_id}').json()
-
-    def get_nodes(
-        self, attributes: dict,
-        offset: Optional[int] = None, limit: Optional[int] = None
-    ) -> Sequence[dict]:
-        params = attributes.copy() if attributes else {}
-        nodes = []
-
+    def get_api_objs(self, params: dict, path: str, limit: int = None,
```

Comment:
This seems like it should be a "private" method e.g. `_get_api_objs()`.

Reply:
Oh, yes. I'll fix it.